### ISSUE TITLE
Make filenames exactly match BOM, or add filenames to the BOM

### ISSUE THREAD

**Issue by WardBenjamin** [mjrice/Wilson2#6]
When assembling the list of parts by filename that I need to print according to the BOM, I found that parts didn't exactly match. What I'm assuming is the Idler bearing guide is marked as a "belt-guide-624", which I only figured out because you use 624rs bearings. Also, some parts are not consistant with the rest, like "servo_mount" instead of "servo-mount" like it should be, and "z-motor-brace-1" and z-motor-brace-2" which are not marked in the BOM, instead it says to print 2 z motor braces and does not specify which files. 

Basically, could you add the correct filenames to the BOM? Assuming you have printed the parts before it should be pretty easy.

**Comment by WardBenjamin** [comment]
Thanks!
